src/index.js

```diff
@@ -0,0 +1,48 @@
+// import _ from 'lodash';
+import './style.css';
+
+const form = document.getElementById('form');
+const tasks = [
+  {
+    description: 'attending mornning session',
+    completed: true,
+    index: 0,
+
+  },
+  {
+    description: 'coding',
+    completed: true,
+    index: 1,
+
+  },
+  {
+    description: 'break dance',
+    completed: false,
+    index: 0,
```

Comment:
Make sure you set the right index for this task.

<img width="868" alt="Screen Shot 2022-07-06 at 18 51 39" src="https://user-images.githubusercontent.com/82426682/177661399-793c746d-388b-4a4f-bc96-1c8909f3dd00.png">